#include "lobby_scene.pb.h"
#include "src/game_logic/thread_local/thread_local_storage.h"
#include "src/network/message_system.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void LobbyService::EnterCrossMainSceneWeightRoundRobin(::google::protobuf::RpcController* controller,
	const ::EnterCrossMainSceneWeightRoundRobinRequest* request,
	::EnterCrossRoomSceneSceneWeightRoundRobinResponse* response,
	 ::google::protobuf::Closure* done)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}
